ios/FluentUI/Core/Colors2.swift

```diff
@@ -0,0 +1,234 @@
+        return colorProvidersMap.object(forKey: theme)?.brandForegroundTint(for: theme) ?? FallbackThemeColor.brandForegroundTint
+    }
+
+    @objc public static func brandForegroundDisabled1(for theme: FluentTheme) -> UIColor {
+        return colorProvidersMap.object(forKey: theme)?.brandForegroundDisabled1(for: theme) ?? FallbackThemeColor.brandForegroundDisabled1
+    }
+
+    @objc public static func brandForegroundDisabled2(for theme: FluentTheme) -> UIColor {
+        return colorProvidersMap.object(forKey: theme)?.brandForegroundDisabled2(for: theme) ?? FallbackThemeColor.brandForegroundDisabled2
+    }
+
+    @objc public static func brandStroke1(for theme: FluentTheme) -> UIColor {
+        return colorProvidersMap.object(forKey: theme)?.brandStroke1(for: theme) ?? FallbackThemeColor.brandStroke1
+    }
+
+    @objc public static func brandStroke1Pressed(for theme: FluentTheme) -> UIColor {
+        return colorProvidersMap.object(forKey: theme)?.brandStroke1Pressed(for: theme) ?? FallbackThemeColor.brandStroke1Pressed
+    }
+
+    @objc public static func brandStroke1Selected(for theme: FluentTheme) -> UIColor {
+        return colorProvidersMap.object(forKey: theme)?.brandStroke1Selected(for: theme) ?? FallbackThemeColor.brandStroke1Selected
+    }
+
+    private static var colorProvidersMap = NSMapTable<FluentTheme, ColorProviding2>(keyOptions: .weakMemory, valueOptions: .weakMemory)
+
+    /// A namespace for holding fallback theme colors (empty enum is an uninhabited type)
+    private enum FallbackThemeColor {
+        static var brandBackground1: UIColor = UIColor(dynamicColor: FluentTheme.shared.aliasTokens.colors[.brandBackground1])
+
+        static var brandBackground1Pressed: UIColor = UIColor(dynamicColor: FluentTheme.shared.aliasTokens.colors[.brandBackground1Pressed])
+
+        static var brandBackground1Selected: UIColor = UIColor(dynamicColor: FluentTheme.shared.aliasTokens.colors[.brandBackground1Selected])
+
+        static var brandBackground2: UIColor = UIColor(dynamicColor: FluentTheme.shared.aliasTokens.colors[.brandBackground2])
+
+        static var brandBackground2Pressed: UIColor = UIColor(dynamicColor: FluentTheme.shared.aliasTokens.colors[.brandBackground2Pressed])
+
+        static var brandBackground2Selected: UIColor = UIColor(dynamicColor: FluentTheme.shared.aliasTokens.colors[.brandBackground2Selected])
+
+        static var brandBackground3: UIColor = UIColor(dynamicColor: FluentTheme.shared.aliasTokens.colors[.brandBackground3])
+
+        static var brandBackgroundTint: UIColor = UIColor(dynamicColor: FluentTheme.shared.aliasTokens.colors[.brandBackgroundTint])
+
+        static var brandForeground1: UIColor = UIColor(dynamicColor: FluentTheme.shared.aliasTokens.colors[.brandForeground1])
+
+        static var brandForeground1Pressed: UIColor = UIColor(dynamicColor: FluentTheme.shared.aliasTokens.colors[.brandForeground1Selected])
```

Comment:
Variable name is `brandForeground1Pressed` but `. brandForeground1Selected ` is used to access value in `colors`.